hooks/command

```diff
@@ -12,8 +12,11 @@ export AWS_ACCESS_KEY_ID=$(echo ${TEMP_ROLE} | jq .Credentials.AccessKeyId | xar
 export AWS_SECRET_ACCESS_KEY=$(echo ${TEMP_ROLE} | jq .Credentials.SecretAccessKey | xargs)
 export AWS_SESSION_TOKEN=$(echo ${TEMP_ROLE} | jq .Credentials.SessionToken | xargs)
 
+
+echo {\"jenkins_job\": \"${JOB}\", \"tag\": \"${BUILDKITE_COMMIT}\"} > ./input.txt
```

Comment:
Just because I'm really not a fan of escaping things where possible, how do you feel about changing this line to:

```
cat << EOF > input.text
{"jenkins_job": "${JOB}", "tag": "${BUILDKITE_COMMIT}"}
EOF
```

Reply:
much nicer